.changeset/tall-buckets-change.md

```diff
@@ -0,0 +1,284 @@
+---
+"@salt-ds/theme": minor
+---
+
```

Comment:
We should add a general note that if teams are customising the theme, they will be impacted and will need to update these customisations.